pkg/webhook/mutation/pod/annotations/annotations.go

```diff
@@ -0,0 +1,21 @@
+package annotations
+
+import "github.com/Dynatrace/dynatrace-operator/pkg/webhook/mutation/pod/mutator"
+
+func SetDynatraceInjectedAnnotation(mutationRequest *mutator.MutationRequest, injectedAnnotation, reasonAnnotation string) {
```

Comment:
package is already called annotations, so these functions don't need to repeat it